utils/check-style/aspell-ignore/en/aspell-dict.txt

```diff
@@ -1602,6 +1602,7 @@ freezed
 fromDaysSinceYearZero
 fromModifiedJulianDay
 fromModifiedJulianDayOrNull
+fromReadableSize
```

Comment:
guess you need to add the other function here too.

Reply:
All flavours added